app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,82 @@
+class TweetController < ApplicationController
+
+
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      @tweets = Tweet.all
+      erb :"/tweets/tweets"
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :"/tweets/create_tweet"
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if logged_in?
+      if params[:content] != ""
+        tweet = Tweet.new(content: params[:content])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```